Makefile

```diff
@@ -134,6 +134,26 @@ else
 	go test -v -timeout 3600s $(TEST_ARGS) ./test/e2e/...
 endif
 
+kind-e2e-test: build kustomize docker-build create-kind-multicluster kind-load-image-multi e2e-test
+
+kind-setup: build kustomize docker-build create-kind-cluster kind-load-image
+	@echo Setting up kind-k8ssandra-0
+	kubectx kind-k8ssandra-0
```

Comment:
good call, I'll use kubectl instead

Reply:
These docs did call for it to be installed as one of the dependancies. I still agree with avoiding it if posssible  since you don't know if it's been installed using krew (call `kubectl ctx`) or standalone (call `kubectx`).

Reply:
It's gone already in the latest version. Turns out I didn't need to switch context in order to setup a single cluster.

Reply:
kubectx, kubens, etc aren't standard tools, but they are fairly well known and widely used. I will gladly advocate for tools that make dev and testing easier and faster. This might be better suited for discussion on the forum or Discord.